.travis.yml

```diff
@@ -16,4 +16,9 @@ addons:
   browserstack:
     username: "chimesdk"
     access_key:
-      secure: "TRGKE6oqaCXAhYcLVu6bu1iH/IvtAIj2cKx7o73ukrOm9ifUfmpAzfVFlQB8evCQM67SP8u2x4QyaWhD+JHZKjUkj/DkY6XjJPyT6VjOhqiZQYU+GGDVI3xwawmap3rWUIW4Gdo2lK4NfTGU1Z+gwVLAmIuNXjgFhaWE93/0ycSS3B4bKUqlGJX/YYZVcpbTnQqS6SOiqt6XJMHiBqIU8+r63RuJcBifrccQbJYz5g5ufQpVagyOScTKKdm8T93vHXp12A3XkMf2W911Ef06XOVRmG/gKET9/FBnM12F6430vDGx8kYeKbo9E7YpRlJS8M6QtO7hpEGfZlZItPWb2cP+tp+W+JRgdnhlO8nSXvhy6RH2iwhLQRP4/B1AOPybab405H0MD+p45jklmkhNw4iY9LXdI9/u3Sd8DuVIkq/bNblVIFKuqw0BFNi5PXgPfizenc3AfPThGJLIUAVUzXte+niF1sajwUB/l3ob0rlg0Nhk1OnsVR6CingN8uzXYo9DAnEzbRgJOazJYASVp5WTP8i4v+9+tMndz3NT1g66FbcY9rFmCTesqMQ7xTjKTIr5/e32wOo67dgolC2orp22tsDmWYtmHZ89NqOmuxFMH3j1oMFCx9tB5aYLth2YTsbe6Iac5d3sND/wbcTZJDEgdpeGqUfmHy3OsM5DHIU="
\ No newline at end of file
+      secure: "TRGKE6oqaCXAhYcLVu6bu1iH/IvtAIj2cKx7o73ukrOm9ifUfmpAzfVFlQB8evCQM67SP8u2x4QyaWhD+JHZKjUkj/DkY6XjJPyT6VjOhqiZQYU+GGDVI3xwawmap3rWUIW4Gdo2lK4NfTGU1Z+gwVLAmIuNXjgFhaWE93/0ycSS3B4bKUqlGJX/YYZVcpbTnQqS6SOiqt6XJMHiBqIU8+r63RuJcBifrccQbJYz5g5ufQpVagyOScTKKdm8T93vHXp12A3XkMf2W911Ef06XOVRmG/gKET9/FBnM12F6430vDGx8kYeKbo9E7YpRlJS8M6QtO7hpEGfZlZItPWb2cP+tp+W+JRgdnhlO8nSXvhy6RH2iwhLQRP4/B1AOPybab405H0MD+p45jklmkhNw4iY9LXdI9/u3Sd8DuVIkq/bNblVIFKuqw0BFNi5PXgPfizenc3AfPThGJLIUAVUzXte+niF1sajwUB/l3ob0rlg0Nhk1OnsVR6CingN8uzXYo9DAnEzbRgJOazJYASVp5WTP8i4v+9+tMndz3NT1g66FbcY9rFmCTesqMQ7xTjKTIr5/e32wOo67dgolC2orp22tsDmWYtmHZ89NqOmuxFMH3j1oMFCx9tB5aYLth2YTsbe6Iac5d3sND/wbcTZJDEgdpeGqUfmHy3OsM5DHIU="
```

Comment:
Can we also clean this up now since we are not running BrowserStack here anymore?

Reply:
sure will remove it